py3status/modules/diskdata.py

```diff
@@ -152,10 +224,10 @@ def _get_free_space(self, disk):
 
         df = self.py3.command_output("df")
         for line in df.splitlines():
-            if (disk and line.startswith(disk)) or (
-                disk is None and line.startswith("/dev/")
+            data = line.split()
+            if self._resolve_disk_name(data[0]) in self._disks or any(
+                data[0].startswith("/dev/" + x) for x in self._disks
```

Comment:
I'm not sure what's going on here, but I wonder maybe it would be better to address this in `post_config_hook` by using a dict of disk paths instead of device names.

Reply:
the problem is, df use full paths, proc/diskstats use device names so in any case we'll have to mangle this data (or storing it twice, with and without prefix). I've removed `startswith` at 229 cause an identity fits better atm

Reply:
Maybe storing it wouldn't be that bad... `alias` key.

Reply:
done